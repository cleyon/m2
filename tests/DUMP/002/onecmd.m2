@newcmd one
@define FOO 1
Begin
@case FOO
Preamble
Of clauses may print in different order - that's okay
@of 1
One
@of 2
Two
@of 3
Three
@otherwise
Not matched
@endcase
End
@endcmd
@one
@dump commands
