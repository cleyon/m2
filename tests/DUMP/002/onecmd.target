Begin
Preamble
One
End
@newcmd one
@define FOO 1
Begin
@case FOO
Preamble
@of 1
One
@of 2
Two
@of 3
Three
@otherwise
Not matched
@endcase
End
@endcmd
