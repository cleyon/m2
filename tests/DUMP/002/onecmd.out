Begin
Preamble
One
End
@newcmd one
@define FOO 1
Begin
@case FOO
Preamble
@of 2
Two
@of 3
Three
@of 1
One
@otherwise
Not matched
@endcase
End
@endcmd
